src/models/Project/Project.ts

```diff
@@ -0,0 +1,12 @@
+export interface Project {
+    id?: string;
+    name?: string;
+    description?: string;
+    location?: {
+        address?: string;
+        city?: string;
+        postalCode?: string;
+        state?: string;
+        country?: string;
```

Comment:
use `Address` type here so it's consistent

Reply:
I got it